test/controllers/game_controller_test.exs

```diff
@@ -0,0 +1,79 @@
+defmodule Yggdrasil.GameControllerTest do
+  use Yggdrasil.ConnCase
+
+  alias Yggdrasil.{Repo, User, Game}
+
+  @user %{username: "tester",
+          password: "password",
+          password_confirmation: "password"}
+
+  @json_api_content_type "application/vnd.api+json"
+  @json_api_content_type_utf8 @json_api_content_type <> "; charset=utf-8"
+
+  setup %{conn: conn} do
+    # need a user to gen a token for api
+    {:ok, user} = %User{}
+    |> User.create_changeset(@user)
+    |> Repo.insert
+
+    #api token
+    {:ok, token, _claims} = Guardian.encode_and_sign user, :token
+
+
+    # set of games to have something to list
+    games = Enum.map 1..10, fn n ->
+      game = Game.changeset(%Game{}, %{"name" => "game#{n}",
+                                       "description" => "game desc #{n}"})
+      {:ok, game} = Repo.insert game
+
+      game
+    end
+
+    conn = conn
+      |> put_req_header("content-type", @json_api_content_type)
+      |> put_req_header("accept", @json_api_content_type)
+
+    {:ok, %{conn: conn, games: games, user: user, token: token}}
+  end
+
+  defp compare_game_lists(resp, games) do
+    games_resp = Poison.decode! resp
+
+    api_games = Enum.map games_resp["data"], fn g ->
+      attrs = g["attributes"]
+      {id, _remainder} = Integer.parse g["id"]
+
+      game = Map.put attrs, "id", id
+      for {key, val} <- game, into: %{}, do: {String.to_atom(key), val}
+    end
+
+    db_games = Enum.map games, fn g ->
+      g
+      |> Map.from_struct
+      |> Map.drop([:inserted_at, :updated_at, :__meta__])
+    end
+
+    db_games == api_games
+  end
+
```

Comment:
@copenhas right now only have two tests that cover the happy case.

right now with only allowing get the only other major one we may want to cover is the case where there are no games, which should return an empty list to the client. I'm sure there are others as well,